roles/o11y_agent/templates/grafana-agent.yaml.j2

```diff
@@ -30,6 +30,13 @@ metrics:
           metrics_path: '/MAAS/metrics'
 {% endif %}
 {% endif %}
+{% if 'maas_postgres' in group_names %}
+    - name: 'postgres'
+      scrape_configs:
+        - job_name: 'database-metrics'
+          static_configs:
+            - targets: ['{{ ansible_hostname }}:9187']
+{% endif %}
 {% endif %}
 
 {% if o11y_loki_url|length > 0 %}
```

Comment:
What about logs? Please add a config to collect postgresql logs. Right now we won't do any special processing of this logs, just throw them at Loki